install_config/upgrading/manual_upgrades.adoc

```diff
@@ -419,10 +419,10 @@ middleware:
 ----
 ====
 
-Or with an environment variable
-`*REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA*`. The environment
-variable is set to `*true*` for the new registry deployments by default.
-Existing deployments need to be modified using:
+Alternatively, use environment variable
+`*REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA*`, which is set to
```

Comment:
use _the_ `X` environment variable.

(it's missing a _the_ but might need a minor arrangement to flow better.